.env.local

```
@@ -1,7 +1,12 @@
-# Required
-NEXT_PUBLIC_SUPABASE_URL=your_supabase_url
-NEXT_PUBLIC_SUPABASE_ANON_KEY=your_supabase_anon_key
-GROQ_API_KEY=your_groq_key
+# Supabase
+NEXT_PUBLIC_SUPABASE_URL=https://...
+SUPABASE_SERVICE_ROLE_KEY=eyJ...
+
+# Upstash QStash
+QSTASH_TOKEN=eyJ...
+QSTASH_CURRENT_SIGNING_KEY=... (optional)
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The QSTASH_CURRENT_SIGNING_KEY key should go before the QSTASH_TOKEN key

(UnorderedKey)